pkg/oc/admin/router/router.go

```diff
@@ -730,13 +753,6 @@ func RunCmdRouter(f *clientcmd.Factory, cmd *cobra.Command, out, errout io.Write
 		},
 	}
 
-	if cfg.StatsPort > 0 && cfg.ExposeMetrics {
-		pc := generateMetricsExporterContainer(cfg, env)
```

Comment:
Think the `generateMetricsExporterContainer` function isn't used anywhere else, if you are removing this code.

Reply:
Removed